packages/code-style/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "@technance/code-style",
     "type": "module",
-    "version": "1.5.0",
+    "version": "1.5.1",
```

Comment:
Wait, we shouldn't bump version like this, let's create a changeset instead.

Revert this version back to `1.5.0` and in the repo's root run `pnpm changeset`, from there pick `patch`.